packages/valory/skills/decision_making_abci/models.py

```diff
@@ -247,6 +247,11 @@ def merge_by_wallet(self):
                 # Add merged user
                 self.data["users"].append(merged_user)
 
+    def reset_period_points(self):
+        """Resets period points"""
+        for u in self.data["users"]:
+            u["current_period_points"] = 0
```

Comment:
It's gotten a bit out of hand how often keys are accessed unsafely in this service.
Creating a structure on retrieval, after performing some validation, would be better because now there are too many places where this might fail, and hard to detect the source if it ever happens.